cluster/src/test/java/com/hartwig/pipeline/tertiary/cuppa/CuppaTest.java

```diff
@@ -73,11 +73,17 @@ protected Stage<CuppaOutput, SomaticRunMetadata> createVictim() {
 
     @Override
     protected List<String> expectedCommands() {
-        return List.of("java -Xmx4G -jar /opt/tools/cuppa/1.7.1/cuppa.jar " + "-categories DNA " + "-ref_data_dir /opt/resources/cuppa/ "
-                        + "-sample_data tumor " + "-sample_data_dir /data/input/results " + "-output_dir /data/output",
-                "/opt/tools/cuppa-chart/1.7.1_venv/bin/python /opt/tools/cuppa-chart/1.7.1/cuppa-chart.py " + "-sample tumor "
-                        + "-sample_data /data/output/" + TUMOR_CUP_DATA_CSV + " -output_dir /data/output",
-                "Rscript /opt/tools/cuppa/1.7.1/CupGenerateReport_pipeline.R tumor /data/output/");
+        return List.of("java -Xmx4G -jar /opt/tools/cuppa/1.7.2/cuppa.jar "
+                        + "-categories DNA "
+                        + "-ref_data_dir /opt/resources/cuppa/ "
+                        + "-sample_data tumor "
+                        + "-sample_data_dir /data/input/results "
+                        + "-output_dir /data/output "
+                        + "-create_pdf",
```

Comment:
Annoyingly the code formatter settings will bring these all back to where they were to start with...

Reply:
Again it is clearer to have them on separate lines, as is done in other tests.